tests/containers/rootless_podman.pm

```diff
@@ -61,6 +62,16 @@ sub run {
     clean_container_host(runtime => $runtime);
 }
 
+sub softfail_and_skip_on_bsc1182874 {
+    assert_script_run("podman images");
+    select_console('root-console');
```

Comment:
why switching to root-console?

Reply:
❯ ll /var/log/messages
-rw-r----- 1 root root 1435539 Jul  2 15:14 /var/log/messages

But i can move the line a liitlte above and avoid this. i didnt think of it because i was started from another place (in common.pm) and i just keep it as it is.

Reply:
MAybe you can move `select_console "user-console";` after this check?

Reply:
i dont think so as you need to check against user. not root. however the PR is not ready from my side.

Reply:
invalid after latest commit